src/pages/transactions.js

```diff
@@ -35,8 +34,7 @@ class TransactionsPage extends React.Component {
   }
 
   async componentDidMount() {
-    const { getLoggedInUser, data } = this.props;
-    const LoggedInUser = await getLoggedInUser();
+    const { LoggedInUser, data } = this.props;
```

Comment:
If component gets updated later (which will happen because `LoggedInUser` is fetched after the mount) then we won't get the updated `LoggedInUser`. We don't need to store the user in the state anymore, we can rely on `this.props` directly.